woocommerce/woocommerce-admin
# Fix attribute filter for "any X" variations.

Fixes #6722.

This PR seeks to fix the "any X" variation case by using the order item meta table for attribute filtering rather than the post meta table.

### Detailed test instructions:

- Create a variable product with variation attributes set to `Any X`, like so:

![Markup on 2021-03-31 at 16:42:27](https://user-images.githubusercontent.com/5512652/113208728-21f1f780-9240-11eb-8fb3-3231d9d3e733.png)

- Place order(s) for that variation
- Verify that you can filter for the "any X" attribute using the attribute filter:

![Markup on 2021-03-31 at 16:42:39](https://user-images.githubusercontent.com/5512652/113208912-62ea0c00-9240-11eb-8326-f15f0ccf152c.png)
